pycaw/magic.py

```diff
@@ -0,0 +1,563 @@
+"""
```

Comment:
I'm sure you will write unit tests for this new module, right? :sweat_smile: :pray:

Reply:
hmm i started writing an example... that should do the trick in general right? ... of course not with msedge ... but similar